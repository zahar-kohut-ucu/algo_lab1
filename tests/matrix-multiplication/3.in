8 8
50 78 5 53 84 3 13 0 
33 61 92 69 1 40 72 54 
29 30 66 65 45 56 30 4 
71 47 91 88 45 76 43 71 
73 97 25 30 100 93 4 16 
42 12 46 74 49 65 69 32 
12 83 29 89 42 27 60 95 
14 98 53 77 45 86 36 98 
8 8
30 71 33 45 68 72 5 61 
39 72 99 50 33 93 66 28 
77 14 80 26 48 62 32 77 
71 19 56 1 21 70 80 97 
10 9 34 65 79 69 60 100 
55 46 97 4 77 33 82 77 
66 86 60 90 81 40 2 73 
31 27 44 55 88 23 22 60 
