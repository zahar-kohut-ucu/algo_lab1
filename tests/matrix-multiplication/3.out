10553 12255 16667 12975 15247 21289 15110 20340 
23988 18833 28962 16671 23865 24094 17327 29174 
17371 12047 21785 10655 18144 19765 16877 25238 
26887 20887 33810 19003 31733 30251 23989 38941 
16903 19041 29057 16927 26788 28424 23953 30417 
20135 16125 23917 15175 24295 21194 17506 30480 
20959 18270 27744 18996 25433 24985 20530 30281 
24384 20358 35060 18884 30064 28435 26374 34778 
